nem.rs

```diff
@@ -76,10 +133,32 @@ fn mnemonic(v: &[String], existing: Vec<&String>) -> String {
 
 fn main() {
     let mut args: Vec<_> = env::args().collect();
-    let contents = fs::read_to_string(".nem.toml").expect("Couldn't open file.");
-    let mut nem_file: NemFile = toml::from_str(&contents).expect("Couldn't parse toml file");
-    nem_file.file_name = String::from(".nem.toml");
-    nem_file.sort();
+
+    let mut nem_files = NemFiles {
+        nem_files: Vec::new(),
+    };
+    let mut path_buf = env::current_dir().expect("Couldn't get CWD");
+    loop {
+        path_buf.push(".nem.toml");
+        let path = path_buf.as_path();
+        if path.exists() {
+            match fs::read_to_string(&path) {
+                Ok(contents) => {
+                    let mut nem_file: NemFile =
+                        toml::from_str(&contents).expect("Couldn't parse toml file");
+                    nem_file.file_name = String::from(path.to_str().unwrap());
+                    nem_file.sort();
+                    nem_files.nem_files.push(nem_file);
+                }
+                Err(_) => {}
```

Comment:
Maybe would be worth `eprintln!`-ing the error? Every `Error` in rust has to also implement `Display`, so you can format it with `"{}"`